### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2949,6 +2949,9 @@ cacluating->calculating
 cacluation->calculation
 cacluations->calculations
 cacluator->calculator
+caclucation->calculation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add caclucate->calculate

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9570,6 +9580,7 @@ guarunty->guaranty
 guassian->gaussian
 Guatamala->Guatemala
 Guatamalan->Guatemalan
+gude->good, guide,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I suspect guide should probably be the first suggestion.

Also add `gud->good`

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: c6c6b2d

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: c6c6b2d

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8752,6 +8755,7 @@ finishs->finishes
 finitel->finite
 finness->finesse
 finnished->finished
+finshed->finished
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add finsh and finshes (also suggest finch/finches for these)

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -13710,6 +13715,7 @@ oen->one
 oerflow->overflow
 ofcource->of course
 offcers->officers
+offcial->official
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add offcials

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -15863,6 +15870,7 @@ quicly->quickly
 quiessent->quiescent
 quinessential->quintessential
 quitely->quite, quietly,
+quith->with
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest quit

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 744ce7e

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 744ce7e

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 744ce7e

**Pull request merged**